modules/ROOT/pages/sap/sap-connector-xml-maven.adoc

```diff
@@ -0,0 +1,105 @@
+= SAP XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule application in XML either from the Studio XML editor 
+or from a text editor, perform the following actions to get access to the connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the 
+canvas, Studio automatically populates the XML code with the connector namespace and schema location.
+
+== Add a Namespace for the Connector
+
+Paste the following code inside the `<mule>` tag of the header of your configuration XML:
+
+[source,xml,linenums]
+----
+http://www.mulesoft.org/schema/mule/sap 
+http://www.mulesoft.org/schema/mule/sap/current/mule-sap.xsd
+----
+
+This example shows how the namespace statements are placed in the `<mule>` XML block: 
+
+[source,xml,linenums]
+----
+<mule xmlns:sap="http://www.mulesoft.org/schema/mule/sap" 
+	xmlns:ee="http://www.mulesoft.org/schema/mule/ee/core"
+	xmlns:http="http://www.mulesoft.org/schema/mule/http"
+	xmlns:file="http://www.mulesoft.org/schema/mule/file" 
+	xmlns="http://www.mulesoft.org/schema/mule/core" 
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation" 
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core 
+	http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+	http://www.mulesoft.org/schema/mule/file 
+	http://www.mulesoft.org/schema/mule/file/current/mule-file.xsd
+	http://www.mulesoft.org/schema/mule/http 
+	http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+	http://www.mulesoft.org/schema/mule/ee/core 
+	http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+	http://www.mulesoft.org/schema/mule/sap 
+	http://www.mulesoft.org/schema/mule/sap/current/mule-sap.xsd">
+----
+
+== Add a POM File Dependency 
+
+The `pom.xml` Maven file generated by Anypoint Studio contains dependencies for a Mule app. When you code a Mule app manually, include this XML snippet in your `pom.xml` file to enable access to this connector. The Mule runtime engine translates the `LATEST` dependency keyword to the current version.
```

Comment:
The first sentence here repeats the statement in the introduction that using Anypoint Studio automatically creates requirements, and that this topic set is about the requirements for manual coding.
If you include this specific comment about the POM file dependency requirement here, suggest adding a similar one to the previous task, to introduce it in a parallel way.

Reply:
Note also that this "Add" task is not presented as steps, just like the previous. Either change the two titles to reflect two concept topics, or create steps to reflect task topics.

Reply:
The second sentence introduces the code snippet, so the snippet should follow it, and the third sentence should move to follow the snippet.